ui/objects/semantic-ui/js/objects/search/components/ResultsListItem.jsx

```diff
@@ -89,54 +122,51 @@ export const ResultsListItemComponent = ({ result, appName }) => {
     "metadata.restorationWork.restorer",
     "<no data>"
   );
-  const desc = _get(
-    result,
-    "metadata.restorationWork.abstract",
-    "<no data>"
-  );
 
   const created = _get(result, "created", "<no data>");
 
-
-
   return (
     <Overridable
       id={buildUID("RecordsResultsListItem.layout", "", appName)}
       result={result}
       title={title}
     >
-      <Grid className="predmety__card" key={result.id}>
-        <Item className="horiz-div predmety__card-content">
-          <Grid className="predmety__card__img-container">
-            
-            <ImageWithFallback src="/static/images/image-noimage.png" fallbackSrc="/static/images/image-404.png"   result={result} classN=''/>
-           
-          </Grid>
-          <Item.Content className="vert-div predmety__card__info">
-            <Grid.Column className="vert-div predmety__card__main-info">
-              <ItemHeader
-                className="predmety__card__title"
-                title={title}
-                searchUrl={searchAppConfig.ui_endpoint}
-                selfLink={`${result.id}`}
-              />
-              <Item.Description className="parag">{restorer}</Item.Description>
-              {/*  {wideScreen && <Item.Description className="parag">{desc.substring(0,70)}...</Item.Description>
-              } */}
-            </Grid.Column>
-            <Item.Group className="horiz-div predmety__card__extra-info">
-              <Item.Extra className="horiz-div predmety__card__extra-info">
-              <p className="parag">Vloženo: {created}</p>
-              </Item.Extra>
-              <DetailsButton
-                className="predmety__card__btn"
-                searchUrl={searchAppConfig.ui_endpoint}
-                selfLink={`${result.id}`}
-              />
-            </Item.Group>
-          </Item.Content>
-        </Item>
-      </Grid>
+      <Item key={result.id}>
+        <ItemImage
+          size="tiny"
+          src={
+            objectImages?.links?.content ?? "/static/images/image-noimage.png"
+          }
+          alt={
+            objectImages?.[0]?.metadata?.caption === undefined
```

Comment:
Please use the same logic for getting an image caption in all places that it appears (in ImgCarousel this is quite different)